grails-app/services/org/pih/warehouse/report/NotificationService.groovy

```diff
@@ -348,5 +351,31 @@ class NotificationService {
         }
     }
 
+    void sendRequisitionStatusNotification(StockMovement stockMovement, Location origin){
+        String TOPIC_ARN = grailsApplication.config.awssdk.sns.order.statusWmsToStocksUpDev
+        String configuredStatusesString = grailsApplication.config?.openboxes?.integration?.requisition?.notification?.configuredStatuses?.toString()
+        List<String> configuredStatuses = []
+        if(configuredStatusesString){
+            configuredStatuses = configuredStatusesString.split(",")
+        }
+        JSONObject notifyJson = null
+        if(configuredStatuses?.contains(stockMovement.status?.toString())){
```

Comment:
```
if (stockMovement.stockMovementStatusCode in configuredStatuses)
```

Reply:
Do we still need this now. Because you said in previous comment that for now send notification for all status

Reply:
Remove it for now. We can add it in if they need us to filter these status change events.